www/class/config-generate/severity.class.php

```diff
@@ -276,4 +297,34 @@ public function getServiceSeverityMappingHostSeverityByName($hc_name)
         $this->service_severity_cache[$hc_name] = &$severity;
         return $severity['sc_id'];
     }
+
+    /**
+     * Export cached objects in corresponding export file
+     */
+    public function generateObjects(): void
+    {
+        foreach ($this->service_severity_cache as $id => $value) {
```

Comment:
you should split in 2 private functions : `generateServiceSeverities` & `generateHostSeverities`